jwst/tso_photometry/tso_photometry.py

```diff
@@ -14,7 +14,9 @@
 log.setLevel(logging.DEBUG)
 
 
-def tso_aperture_photometry(datamodel, xcenter, ycenter, radius, radius_inner, radius_outer):
+def tso_aperture_photometry(
+    datamodel, xcenter, ycenter, radius, radius_inner, radius_outer, gain_2d
+):
     """
     Create a photometric catalog for NIRCam TSO imaging observations.
```

Comment:
Any reason this says NIRCam TSO imaging in particular?

Reply:
No, that dates back to Oct 2016 when the very first version of this file was made. I'll update that